vdslib/src/tests/distribution/distributiontest.cpp

```diff
@@ -434,7 +434,8 @@ TEST(DistributionTest, test_distribution)
                 _distribution[i].second = distr.getIdealStorageNodes(
                         systemState, document::BucketId(26, i));
                 sort(_distribution[i].second.begin(), _distribution[i].second.end());
-                unique(_distribution[i].second.begin(), _distribution[i].second.end());
+                auto unique_nodes = unique(_distribution[i].second.begin(), _distribution[i].second.end()) - _distribution[i].second.begin();
```

Comment:
This might benefit from using `std::distance` instead, to make the semantics more clear

Reply:
Done.